.travis.yml

```diff
@@ -81,7 +81,7 @@ matrix:
           services:
             - docker
         - os: osx
-          osx_image: xcode8
+          osx_image: xcode9
```

Comment:
Xcode 9 is still in beta, latest from 8 line, 8.3, doesn't support C++17 at all.